examples/pom.xml

```diff
@@ -138,6 +138,18 @@
                         </exclusion>
                     </exclusions>
                 </dependency>
+
+                <dependency>
```

Comment:
Please add the following dependency

  `<dependency>
                     <groupId>org.apache.ignite</groupId>
                     <artifactId>ignite-spark-2.10</artifactId>
                     <version>${project.version}</version>
                 </dependency>`

to <id>scala-2.10</id> profile below.